pl/tutorials/tour/case-classes.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: tutorial
+title: Klasy case
+
+disqus: true
+
+tutorial: scala-tour
+num: 10
+language: pl
+tutorial-next: pattern-matching
+tutorial-previous: currying
+---
+
+Scala wspiera mechanizm _klas case_. Klasy case są zwykłymi klasami z dodatkowymi założeniami:
```

Comment:
Right, `klasy przypadków` sounds more natural. I wasn't sure if I should translate terms that are unique for Scala, but if it was translated before then I guess we should use this term.